throughput/src/bin/t_pubsub_thr.rs

```diff
@@ -108,24 +112,38 @@ impl TransportPeerEventHandler for MyMH {
     }
 }
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 #[structopt(name = "s_pubsub_thr")]
 struct Opt {
-    #[structopt(short = "l", long = "listen")]
+    /// which endpoints to listen on. `zenohd` will add `tcp/0.0.0.0:7447` to these locators if left empty,
```

Comment:
Comment only. Is the description of "`zenohd` will add `tcp/0.0.0.0:7447` to these locators if left empty" necessary? Or, say, is it only related to `zenohd`?

Reply:
I just copied the documentation from zenoh source code. It may be a good idea to remove them. Thanks!

Reply:
@eddie9712 I just made a new PR, can you help me review it and merge it?

Reply:
I'm confident with the change @YuanYuYuan made. To make the PR done asap without further waiting, I'll do the merge.